mne/epochs.py

```diff
@@ -1170,7 +1170,7 @@ def get_data(self):
         data : array of shape (n_epochs, n_channels, n_times)
             A copy of the epochs data.
         """
-        return self._get_data()
+        return self._get_data().copy()
```

Comment:
in fact it was on purpose but the docstring was not correct.

my feeling is that the automatic copy can be problematic for large epochs objects